MANIFEST.in

```diff
@@ -1 +1,19 @@
 include gluonts/model/r_forecast/R/*.R
+
+prune .devtools
+prune .github
+prune examples
+prune evaluations
+prune docs
+prune test
+
+
+prune src/gluonts/nursery
```

Comment:
I've included the nursery to be excluded.

I was thinking that we could have a second package, just for the nursery, if we want to have it at all.